main.py

```diff
@@ -1,10 +1,37 @@
 import init_django_orm  # noqa: F401
+import json
```

Comment:
`import json` utilizes the standard library, place it in the first place according to [PEP-8](https://peps.python.org/pep-0008/#imports).